tensorflow/c/c_api_test.cc

```diff
@@ -1543,6 +1543,33 @@ TEST(CAPI, TestBitcastFrom_Reshape) {
   TF_DeleteTensor(b);
 }
 
+TEST(CAPI, TestFromProto) {
+  Tensor t_cc(DT_FLOAT, TensorShape({2, 3}));
+  t_cc.flat<float>().setConstant(1.0);
+  tensorflow::TensorProto t_proto;
+  t_cc.AsProtoField(&t_proto);
+
+  TF_Buffer* t_buffer = TF_NewBuffer();
+  tensorflow::MessageToBuffer(t_proto, t_buffer);
```

Comment:
This returns a `Status` which cannot be ignored. Please change to:

```
  Status s = tensorflow::MessageToBuffer(t_proto, t_buffer);
  EXPECT_TRUE(s.ok()) << s.error_message();
```

Reply:
Done